src/main/java/com/hackerramp/subscription/controller/OrdersController.java

```diff
@@ -0,0 +1,41 @@
+package com.hackerramp.subscription.controller;
+
+import com.hackerramp.subscription.services.entities.OrderDetailsResponse;
+import com.hackerramp.subscription.services.entities.OrderRequest;
+import com.sun.istack.internal.NotNull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import com.hackerramp.subscription.db.OrdersRepo;
+import com.hackerramp.subscription.db.entities.Orders;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/orders")
+public class OrdersController {
+
+    @Autowired
+    private OrdersRepo ordersRepo;
+
+    @GetMapping("/showAllOrders")
+    public ResponseEntity<List<Orders>> getEmployees(){
+        return ResponseEntity.ok().body((List<Orders>) ordersRepo.findAll());
+    }
+
+    @GetMapping(value = {"/uidx/{uidx}"})
+    private OrderDetailsResponse getOrdersByUidx(@PathVariable("uidx") @NotNull String uidx) throws Exception{
+        return null;
+    }
+
+    @GetMapping(value = {"/subscriptionId/{orderId}"})
```

Comment:
Here also, make it as subscriptions/{orderId}. Here subscription and order doesn't go well.